splunk_add_on_ucc_framework/data_ui_generator.py

```diff
@@ -34,15 +33,15 @@ def generate_nav_default_xml(
     include_inputs: bool,
     include_dashboard: bool,
     include_configuration: bool,
-    default_view: str,
+    default_view: Union[str, None],
```

Comment:
I was wondering about this syntax, I found https://google.github.io/styleguide/pyguide.html#3195-nonetype

Reply:
Made the changes.